marlin/src/marlin/errors.rs

```diff
@@ -61,6 +61,15 @@ impl From<snarkvm_polycommit::Error> for MarlinError {
     }
 }
 
+impl MarlinError {
+    pub fn into_snark_error(&self, prefix: &str) -> SNARKError {
```

Comment:
Is there a reason why we implemented a standalone method to convert this error?
The `from` method is implemented for `SNARKError` just below.

Reply:
all of these changes you pointed out are from commit #364 which already exist's on master but got merged into ir-updates while i was updating from master yesterday. so im not sure